amy/settings.py

```diff
@@ -338,6 +338,17 @@
     },
 }
 
+# settings for PyData application
+ENABLE_PYDATA = os.environ.get('AMY_ENABLE_PYDATA', False)
+
+if ENABLE_PYDATA:
+    INSTALLED_APPS.insert(
+        INSTALLED_APPS.index('workshops'),
+        'pydata'
+    )
+    PYDATA_USERNAME_SECRET = os.environ.get('AMY_PYDATA_USERNAME')
+    PYDATA_PASSWORD_SECRET = os.environ.get('AMY_PYDATA_PASSWORD')
```

Comment:
Are these two settings optional? That is, is `None` valid value for both username and password?